torch/_decomp/decompositions.py

```diff
@@ -4232,6 +4232,100 @@ def squeeze_default(self: Tensor, dim: Optional[int] = None):
         return aten.squeeze.dims(self, [dim])
 
 
+@register_decomposition(
+    [aten.max.default, aten.max.unary_out, aten.max.dim, aten.max.dim_max]
+)
+def unary_max(
```

Comment:
moving these back to `decompositions.py`. Copy pasting from the other comment thread:

> @peterbell10 so running into some slight issues with renaming unary_max to max. There already is an [existing refs implementation for maximum](https://github.com/pytorch/pytorch/blob/5183760ca52ea7dbd892ebc88cabdd43ebc81cbe/torch/_refs/__init__.py#L1641-L1642). If I rename unary_max to max, then the test cases for maximum will try to call _refs.max instead of _refs.maximum, which will fail because the function signatures are completely different. I'm not sure why this happens.

> One workaround I can think of is to keep the unary_max name for the function, and add PythonRefInfo entries registered to _refs.unary_max accordingly. Is this an acceptable solution?

> Hmm, now running into issues with torch.argmax not having a refs implementation, so the tests for unary_max will fail as well.

In light of these challenges, I think it may be more appropriate to just move these decomps back to decompositions.py, since we are unable to test the decomposition as a reference implementation. What do you think?